tsconfig.json

```diff
@@ -11,6 +11,5 @@
     "moduleResolution": "node",
     "baseUrl": "./src",
     "esModuleInterop": true
-  },
-  "include": ["src/**/*", "tests/**/*"]
```

Comment:
Why removing this part ? Do you plan to extend this config in each package instead ?
If no, we could do `"include": ["packages/*/src/**/*", "packages/*/tests/**/*"]`

Reply:
right..the prev doesn't work well 
new one is good

Reply:
still, it's wrong...because you will get:
```packages
  /renderer
    /lib
       /renderer
       /workspace
       /templace
```

Reply:
we need to rewrite `include` option in each of package..to make it work correctly

Reply:
So, don't forget to extend this file and to add `"include": ...` in every package

Reply:
Yes, I've done it